x-pack/plugins/security_solution/public/common/components/query_bar/index.test.tsx

```diff
@@ -187,13 +186,9 @@ describe('QueryBar ', () => {
 
   describe('state', () => {
     test('clears draftQuery when filterQueryDraft has been cleared', () => {
-      const KibanaWithStorageProvider = createKibanaContextProviderMock();
-
       const Proxy = (props: QueryBarComponentProps) => (
         <TestProviders>
-          <KibanaWithStorageProvider services={{ storage: { get: jest.fn() } }}>
```

Comment:
I think these tests must have been written before our kibana_react mocks (used by `TestProviders`) included `storage` mocks; this code has been unnecessary since that was added.